dicomexporter/exporter.py

```diff
@@ -13,6 +13,31 @@
 # from helpers.itk import convertITKTypeToVTKType, getMetadata, getMetadataList
 # from helpers.volume import VolumeData
 
+class IterableEnum(type):
+    def __iter__(self):
+        for attr in dir(self):
+            if not attr.startswith('__'):
+                yield attr
+
+
+class ALLOWED_EXTENSIONS(metaclass=IterableEnum):
+    vti = 'vti'
+    vtkjs = 'vtkjs'
+
+
+def splitext(path):
+    """
+    Given a path, return it without extensions and a list of all its
+    extensions (without a leading dot '.').

```

Comment:
Why a given path would have several extensions ?

Reply:
Refering to what @finetjul wrote:

> To make things robust, please consider that some file formats may have multiple file_extensions (e.g. mha, mhd)